homeassistant/components/fronius/sensor.py

```diff
@@ -641,6 +641,10 @@ class FroniusSensorEntityDescription(SensorEntityDescription):
         native_unit_of_measurement=PERCENTAGE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    FroniusSensorEntityDescription(
+        key="backup_mode",
```

Comment:
Like already mentioned this should be a `binary_sensor` and not a `sensor`.

Reply:
Yeah, it's not quite as simple as just making this a subclass of a BinarySensorEntityDescription though, there's some assumptions that are made in the code that generates the actual SensorEntity instances that they are all one class.